### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/qt/server/server.go [hunk 1]
```diff
@@ -130,6 +130,9 @@ func serve(
 			},
 			DeviceInfosFunc: usb.DeviceInfos,
 			SystemOpenFunc:  system.Open,
+			UsingMobileData: func() bool {
+				return false
```
Comment (x1ddos) [inline frontends/qt/server/server.go:134]: I suppose a user could also be on a tethered connection over a phone but there's not much we can do, right.

**Comment by x1ddos** [comment]
Looks good but:

```
cmd/servewallet/main.go:119:20: cannot use webdevEnvironment literal (type webdevEnvironment) as type backend.Environment in argument to backend.NewBackend:

	webdevEnvironment does not implement backend.Environment (missing UsingMobileData method)

# github.com/digitalbitbox/bitbox-wallet-app/frontends/qt/server

frontends/qt/server/server.go:133:4: unknown field 'UsingMobileData' in struct literal of type bridgecommon.BackendEnvironment
```

**Comment by benma** [comment]
> Looks good but:

yep fixed up CI now hopefully.

File: frontends/qt/server/server.go (hunk 1 above)
Comment (benma) [reply frontends/qt/server/server.go:134]: I guess not.

**Pull request merged**